docs/integrations/mirroring.md

```diff
@@ -90,9 +90,9 @@ def get_remote_data_command(client, args):
 * **GetModifiedRemoteDataArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
 Arguments explanation:
   - *last_update* - Date string represents the last time we retrieved modified incidents for this integration.
-* **GetModifiedRemoteDataResponse** - this is the object that maintains the format in which you should order the results from this function. You should use `return_results` on this object to make it work. 
+* **GetModifiedRemoteDataResponse** - this is the object that maintains the format in which you should order the results from this function. You should use `return_results` on this object to make it work.
 Arguments explanation. **You must provide one of the following:**
-  - *modified_incident_ids* - a list of incidents that were modified since the last check. Later `get-remote-data` command will run on only modified incidents.
+  - *modified_incident_ids* - a list of the IDs of the incidents that were modified since the last check. IDs must be of type string. Later `get-remote-data` command will run on only modified incidents.
```

Comment:
```suggestion
  - *modified_incident_ids* - a list of strings representing incident IDs that were modified since the last check. Later `get-remote-data` command will run on only modified incidents.
```